drivers/sensor/max30208/max30208.c

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright (c) 2021, Silvano Cortesi
+ * Copyright (c) 2021, ETH Zürich
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT maxim_max30208
+
+#include <sys/byteorder.h>
+#include <logging/log.h>
+
+#include "max30208.h"
+
+LOG_MODULE_REGISTER(MAX30208, CONFIG_SENSOR_LOG_LEVEL);
+
+static int max30208_start_measurement(const struct device *dev)
+{
+	const struct max30208_config *config = dev->config;
+
+	/* Trigger one measurement */
+	int ret = i2c_reg_write_byte_dt(&config->bus, MAX30208_REG_TEMP_SETUP,
+					MAX30208_CONVERT_T_MASK);
+
+	if (ret < 0) {
+		return ret;
+	}
+
+	return 0;
+}
+
+static int max30208_channel_get(const struct device *dev, enum sensor_channel chan,
+				struct sensor_value *val)
+{
+	if (chan != SENSOR_CHAN_AMBIENT_TEMP) {
```

Comment:
`SENSOR_CHAN_ALL` should also be supported, see https://github.com/zephyrproject-rtos/zephyr/blob/main/include/drivers/sensor.h#L513

Reply:
but this is channel get, shouldn't it be specified here?